server/services/BarChartTableService/index.js

```diff
@@ -33,36 +30,30 @@ class BarChartTableService {
 
   csvTableData = () => {
     return {
-      tableColumns: this.websiteColumns.map(({ name }) => name),
-      tableRows: this.websiteRows.map((siteData) =>
-        siteData.map(({ data }) => data)
-      ),
+      tableColumns: [SITE, ...this.websiteColumns.map(({ name }) => name)],
+      tableRows: this.websiteRows.map((siteData) => Object.values(siteData)),
```

Comment:
is the data in the rows going to be off by 1 since we add the SITE column first before the website columns on the line above?

Reply:
You bring up a good point that this could be much more accurate, it will definitely not be off by one because this first header is in the FE portion of the diff.

Good catch, I updated the logic for accuracy. 

[69622e9](https://github.com/AMP-SCZ/dpdash/pull/695/commits/69622e927bf33326b910d3e57f1df4ef08035785)

Reply:
it still feels weird to have the columns and rows not match up. Why is the extra row in the front-end and the extra column in the back-end?

Reply:
[Subject Counts (copy) example.csv](https://github.com/AMP-SCZ/dpdash/files/14655153/Subject.Counts.copy.example.csv)